server/src/main/java/org/apache/iotdb/db/query/executor/EngineQueryRouter.java

```diff
@@ -139,7 +139,7 @@ public QueryDataSet fill(List<Path> fillPaths, long queryTime, Map<TSDataType, I
       QueryContext context)
       throws StorageEngineException, QueryProcessException, IOException {
 
-    long nextJobId = context.getJobId();
+    long nextJobId = context.getQueryId();
```

Comment:
Maybe it's better to unify `queryId` and `jobId`. It looks like confused if you mix using them.